gprofiler/main.py

```diff
@@ -83,6 +83,7 @@ def __init__(
         client: APIClient,
         collect_metrics: bool,
         collect_metadata: bool,
+        separate_applications: bool,
```

Comment:
Rename it per the new name we have for it: `add_application_ids`?

Reply:
Changed to `identify_applications` which suits good IMO